tool/stats_lib.dart

```diff
@@ -94,6 +96,9 @@ class CommonMarkTestCase {
       json['end_line'] as int,
       json['markdown'] as String /*!*/,
       json['html'] as String,
+      json['extensions'] == null
```

Comment:
This is a lot of overhead and special casing for this one extension. It would be better if this could all be done like the other extensions.